core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -528,6 +528,12 @@ def fit(self, **kwargs):
             logger.warning(f'\tWarning: Model has no time left to train, skipping model... (Time Left = {round(kwargs["time_limit"], 1)}s)')
```

Comment:
After model has been trained with feature-pruning, will `AbstractModel.get_info()` show the remaining features needed for prediction, or the full list of features used during training?  Might be good to store both in this object.

Reply:
It shows the remaining feature only, since we fit a copy of the original model on the pruned set of features. In my opinion just having the remaining features should be fine, but I can add this in if you feel otherwise.